archetypes/blog-post.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog
+title: "{{ replace .Name "-" " " | title }}"
+date: {{ .Date }}
+draft: true
+slug: <seo-friendly-version-of-title-separated-by-dashes>
+---
+
+**Author:** <your name> (<your organization name>), <another author's name> (<their organization>)
+
+<!--  
+Instructions:
+- Replace these instructions and the following text with your content.
+- Replace `<angle bracket placeholders>` with actual values. For example, you would update `date: <yyyy>-<mm>-<dd>` to look something like `date: 2021-10-21`.
+- For convenience, use third-party tools to author and collaborate on your content.
+- To save time and effort in reviews, check your content's spelling, grammar, and style before contributing.
+- Feel free to ask for assistance in the Kubernetes Slack channel, [#sig-docs-blog](https://kubernetes.slack.com/archives/CJDHVD54J).
+-->
```

Comment:
Hi @rolfedh .
Because this is an archetype, I would expect to use this file without making too many (or any) changes.
If there are headings that should appear in every blog post, then this is a good place to "enforce" that structure.
I would not make users delete sections of the page content. You could provide a link (in the comment) to the sample blog page in the contribute section?
The front matter and any "required" headings or page elements would be created when the page is initialized.

Reply:
I hear you, @kbhawkey. How might we best solve the issue that gave rise to this PR: 

"Blog post contributors have issues creating new blog posts. They sometimes miss basic formatting and style recommendations.

Potential contributors might give up without trying if they have trouble finding or understanding our guidance.

Providing a template that embodies our best practices might encourage contributors, make their work easier, and produce better results."

I would add that, often, contributors forge ahead without following guidance. This wastes effort and saps goodwill during the review cycle.

What are the different ways we can solve this issue, and how might we apply those solutions to this PR?

Reply:
The [hugo doc's topic for archetypes](https://gohugo.io/content-management/archetypes/) mentions: "Archetypes are content template files in the archetypes directory of your project that contain preconfigured front matter _and possibly also a content disposition_ for your website’s content types. These will be used when you run hugo new." (emphasis mine)

Reply:
I was taking inspiration from [these modular docs templates](https://github.com/redhat-documentation/modular-docs/tree/master/modular-docs-manual/files) we use at Red Hat, which provide a mixture of required elements, instructions, and example content.

Reply:
What do you think @kbhawkey: what information can we incorporate to help contributors, and what should we remove?

Reply:
> Hi @rolfedh .
> Is there more work to do?
> Holding, if there are any other comments or suggestions. If not, feel free to unhold.
> Thanks.
> /approve
> /hold

I'll trim the content as discussed and put this up for merge later today.

Reply:
Hi, @kbhawkey. Thanks for your review. I've updated the content based on our conversation and pushed the changes. Any suggestions? If not, this is ready to merge.

Reply:
/unhold